security.md

```diff
@@ -0,0 +1,15 @@
+# Security
+
+Security is at the heart of a healthy and dependable software ecosystem, and therefore also at the heart of the SSWG. The documents linked from this page intend to provide clear and simple guidance for both users and developers of packages listed in the SSWG's package index.
```

Comment:
Normally I'd advise _away_ from passive voice, but in this case it sounds better to me 🤷‍♀️ 
```suggestion
Security is at the heart of a healthy and dependable software ecosystem, and therefore also at the heart of the SSWG. The documents linked from this page are intended to provide clear and simple guidance for both users and developers of packages listed in the SSWG's package index.
```